sdk/storage/azblob/zt_examples_test.go

```diff
@@ -52,76 +37,72 @@ func Example() {
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// All operations in the Azure Storage Blob SDK for Go operate on a context.Context, allowing you to control cancellation/timeout.
-	ctx := context.Background() // This example has no expiry.
-
-	// This example showcases several common operations to help you get started, such as:
-
 	// ===== 1. Creating a container =====
 
-	// First, branch off of the service client and create a container client.
+	// First, create a container client, and use the Create method to create a new container in your account
 	container := service.NewContainerClient("myContainer")
-	// Then, fire off a create operation on the container client.
-	// Note that, all service-side requests have an options bag attached, allowing you to specify things like metadata, public access types, etc.
-	// Specifying nil omits all options.
-	_, err = container.Create(ctx, nil)
+	// All functions that make service requests have an options struct as the final parameter.
+	// The options struct allows you to specify optional parameters such as metadata, public access types, etc.
+	// If you want to use the default options, pass in nil.
+	_, err = container.Create(context.TODO(), nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	// ===== 2. Uploading/downloading a block blob =====
-	// We'll specify our data up-front, rather than reading a file for simplicity's sake.
 	data := "Hello world!"
 
-	// Branch off of the container into a block blob client
+	// Create a new BlockBlobClient from the ContainerClient
 	blockBlob := container.NewBlockBlobClient("HelloWorld.txt")
 
 	// Upload data to the block blob
-	_, err = blockBlob.Upload(ctx, NopCloser(strings.NewReader(data)), nil)
+	_, err = blockBlob.Upload(context.TODO(), streaming.NopCloser(strings.NewReader(data)), nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	// Download the blob's contents and ensure that the download worked properly
-	get, err := blockBlob.Download(ctx, nil)
+	get, err := blockBlob.Download(context.TODO(), nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// Open a buffer, reader, and then download!
+	// Use the bytes.Buffer object to read the downloaded data.
 	downloadedData := &bytes.Buffer{}
-	reader := get.Body(RetryReaderOptions{}) // RetryReaderOptions has a lot of in-depth tuning abilities, but for the sake of simplicity, we'll omit those here.
+	reader := get.Body(azblob.RetryReaderOptions{}) // RetryReaderOptions options for tuning downloads.
 	_, err = downloadedData.ReadFrom(reader)
 	if err != nil {
 		return
 	}
+
 	err = reader.Close()
 	if err != nil {
 		return
 	}
 	if data != downloadedData.String() {
-		log.Fatal("downloaded data doesn't match uploaded data")
+		log.Fatal("downloaded data does not match uploaded data")
+	} else {
+		fmt.Printf("Downloaded data: %s\n", downloadedData.String())
 	}
 
 	// ===== 3. list blobs =====
```

Comment:
Nitpick for consistency with the other steps.
```suggestion
	// ===== 3. Listing blobs =====
```